.tugboat/steps/2-update.sh

```diff
@@ -10,7 +10,7 @@ echo "Updating..."
 mv .tugboat .tugboat-tmp
 #drainpipe-end
 composer install
-./vendor/bin/task sync
+./vendor/bin/task tugboat:sync
```

Comment:
This would probably break projects that have already customized the `sync` command.

@deviantintegral Do we move forward with this as a breaking change and make a release note that people should update their projects?

Reply:
Yeah, I don't like the idea of breaking existing projects if we can avoid it.

What if we did something like check if `tugboat:sync` exists, and then fall back to sync otherwise?